index.html

```diff
@@ -23,25 +23,160 @@
     </header>
 
 
-    <section class="container">
+    <section id="about">
 
-    <h1 class="primary-text">
+    <h1 class="title">
             I’m Miguel Cuello.
             Glad to see you!
-        </h1>
-        <p class="first-text">
+    </h1>
+        <p class="text">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-        <h3>LET’S CONNECT</h3>
-        <ul id="social">
-            <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
-            <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
-            <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
-            <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+
+    <p class="project">LET’S CONNECT</p>
+
+    <ul class="social">
+        <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
+        <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
+         <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+        <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+        <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+    </ul>
+
+    </section>
+
+    <section  class="card" id="tonic">
```

Comment:
- [ ] your web design doesnt match the figma design in terms of aligning the contentrs of each card as shown below please kindly make sure you try to replicate the figma design as it is important to show that you have a good eye for details and can confidently reproduce any given figma design. :+1: 
- figma design
![align-w](https://user-images.githubusercontent.com/89055502/235588445-b188b74e-3044-45fb-a618-9947e027f2e7.png)
- your design
![align-s](https://user-images.githubusercontent.com/89055502/235588537-b07b44e9-6f5a-4287-b6d6-fc7f83297407.png)